src/chrome/content/rules/MERY.jp.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+
+	Problematic subdomains:
+
+		- assets	(akamai)
+		- bravo	(akamai)
+		- cashmere	(akamai)
+		- content	(akamai)
+		- galena	(akamai)
```

Comment:
I see, no problem. Maybe we'll have to take some time to enable the domains which have been excluded due to the case ;-)